### PR TITLE
Add helm release action

### PR DESCRIPTION
**Pull request by VidarHUN**
- Tested locally with `act`. 
- Triggered only if the `l7mp-operator` or `l7mp-ingress` has changes in `Chart.yml`.
- A real test should be done.
- If it is ok, the #86 should be closed.

### CODE REVIEW

File: .github/workflows/helm.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: 'Helm release'
+
+on:
+  push:
+    paths:
+      - 'helm-charts/l7mp-operator/Chart.yml'
+      - 'helm-charts/l7mp-ingress/Chart.yml'
+
```
Comment (nemethf) [inline .github/workflows/helm.yml:8]: can it have an additional condition similar to the folowing?
```
on:
  push:
    branches: [ master ]
```

File: .github/workflows/helm.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        uses: actions/checkout@v2
+      - name: 'pyhton_client generate'
+        shell: bash
+        run: .python-client/generate
+      - name: 'k8s-operator build'
+        shell: bash
+        run: .k8s-operator/build
+      - name: 'build helm charts'
+        run: |
+          cd helm-charts
```
Comment (nemethf) [inline .github/workflows/helm.yml:25]: I think, at this point, `l7mp/helm-charts/l7mp-operator/crds/crds.yml` should point to `l7mp/k8s-operator/crd.yaml`, which is a generated file and therefore it's not in the git repo.  `l7mp/k8s-operator/build` generates `crd.yaml`, but it does much more.  Maybe it's enough to call `l7mp/openapi/convert-schema -i ./crd.template.yml -o crd.yml -t k8s`, I'm not sure.

File: .github/workflows/helm.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          helm package l7mp-operator l7mp-ingress
+          cd ..
+      - name: 'update l7mp.io/charts'
+        run: |
+          git clone https://github.com/l7mp/l7mp.io.git
+          copy helm-charts/*.tgz l7mp.io/charts
+          helm repo index l7mp.io/charts/ --url https://l7mp.io/charts
+          git add l7mp.io
+          git commit -m "Automatic publish from l7mp/l7mp Helm release action"
+          git push l7mp.io master
```
Comment (nemethf) [inline .github/workflows/helm.yml:35]: Commit message should be something like the following:
```
git commit -m "Update helm charts from l7mp/l7mp"  -m "(triggered by the 'Helm release' github action.)"
```
See, for example, https://chris.beams.io/posts/git-commit/

**Review by nemethf — CHANGES_REQUESTED** [review]
Thanks for working on it

File: .github/workflows/helm.yml (hunk 1 above)
Comment (VidarHUN) [reply .github/workflows/helm.yml:9]: Please see the changes in [40ee500](https://github.com/l7mp/l7mp/pull/99/commits/40ee500edcddf51e984fbea55d0cd4c255bbe4ba).

File: .github/workflows/helm.yml (hunk 2 above)
Comment (VidarHUN) [reply .github/workflows/helm.yml:26]: Please see the changes in [40ee500](https://github.com/l7mp/l7mp/pull/99/commits/40ee500edcddf51e984fbea55d0cd4c255bbe4ba).

File: .github/workflows/helm.yml (hunk 3 above)
Comment (VidarHUN) [reply .github/workflows/helm.yml]: Please see the changes in [40ee500](https://github.com/l7mp/l7mp/pull/99/commits/40ee500edcddf51e984fbea55d0cd4c255bbe4ba).

**Pull request merged**